app/client/reducers/organisation/reducers.ts

```diff
@@ -0,0 +1,50 @@
+import {combineReducers} from "redux";
```

Comment:
for this and the other reducer i added, i changed up how the files are organized, but i used the same reducer+saga format as the other reducers. should i be changing it to something else? you said there was a better format in the mobile code?

Reply:
oh that's relating to how the Sagas update all the objects - so this is fine as it is right now!